src/main/game/game.ts

```
@@ -93,35 +99,35 @@ export class GameAPI {
                 detached: true,
             });
 
-            this.gameProcess.stdout.on("data", (data) => {
+            this.gameProcess?.stdout?.on("data", (data) => {
```

if we don't have a gameProcess defined, something is very wrong. This function should throw an error instead of using `this.gameProcess?`